src/components/DefinitionEntry.js

```diff
@@ -91,6 +103,26 @@ export default class DefinitionEntry extends React.Component {
     return licenseExpression ? <span>{licenseExpression}</span> : <span>&nbsp;</span>
   }
 
+  getRevisionUrl(definition) {
+    const coordinate = get(definition, 'coordinates')
+    switch (coordinate.provider) {
+      case 'github':
+        return ''
+      case 'npmjs':
+        return `${this.getLocationURL(definition)}/v/${coordinate.revision}`
+    }
+  }
+
+  getLocationURL(definition) {
+    const coordinate = get(definition, 'coordinates')
+    switch (coordinate.provider) {
+      case 'github':
+        return `https://${coordinate.provider}.com/${coordinate.namespace}/${coordinate.name}`
```

Comment:
strictly speaking the `provider` value is NOT the name of the website. It is a symbolic label that just happens to frequently (but not always) match up. Just hardcode github.com and npmjs.com etc here